config/alphagov.yml

```diff
@@ -217,6 +217,11 @@ govuk-data-labs:
     - WIP
     - "[WIP]"
 
+  exclude_labels:
+    - "wip"
+    - "WIP"
+    - "ignore"
```

Comment:
I think you also need to set a boolean attribute of `use_labels: true` to the team config. As per: https://github.com/alphagov/seal/blob/8cbedd271e7a707f06b1323b36f1e7c68c507777/lib/team.rb#L14

Reply:
ohhhh I misunderstood our convo

Reply:
@kevindew [fixed](https://github.com/alphagov/seal/pull/198/commits/7e841e238c44d995d3b6ee1e15609d53d8ece415)

Reply:
Nice one, nit that it's a bit unusual to have the true on a new line. Compare it to say: https://github.com/alphagov/seal/blob/7e841e238c44d995d3b6ee1e15609d53d8ece415/config/alphagov.yml#L171

I was unsure whether it was valid YAML or not.

Reply:
@kevindew it's [still valid](https://codebeautify.org/yaml-validator/cbaedb9d) but for consistency wit the other `true` I've amended it.